src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -270,6 +275,28 @@ public boolean hasPublicMember(GHUser user) {
         }
     }
 
+    /**
+     * Returns {@code true} if this organization can use projects.
+     *
+     * <p>
+     * This flag doesn't guarantee the actual presence of any projects.
```

Comment:
Please move `<p>` to this line. It's here and below.

Reply:
Unfortunately, but this formatting is configured for the module. It just doesn't pass the build if I don't follow. Maven `spotless-check` fails. Auto-fix leads to what we see. Such formatting is used throughout the module.

Reply:
I've disabled this check and fixed formatting.